### PR TITLE
Feature/access name of current repo

### PR DESCRIPTION
**Pull request by hlinero**
# Background
I am interested  in accessing the full name of the repository where the workflow is executing. As discussed in issue #4, one way of doing so is by creating a custom query; however, I found that approach too complex. 

# Scope
The purpose of this PR is to provide access to the full name of the repository where this workflow is executing without the need of creating a custom query.

With this new implementation we use the default environment variable `GITHUB_REPOSITORY` to accomplish the goal mentioned above.

From the template file, the user can then use the variable `{{CURRENT_REPO_FULL_NAME }}` to access the value in `GITHUB_REPOSITORY`.

# Notes
I could not find a contribution guideline so I made the following assumptions:
- I used [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) as the standard for writing commit messages.
- I tried my best to follow the naming convention based on the existing code.

While installing the project, the file `package-lock.json` was automatically changed. I am not sure if the changes to this file should be included or not, please let me know.

### CODE REVIEW

**Comment by probablykasper** [comment]
Just realized I screwed up, sorry. I don't think you can get the current repo name using a custom query without specifying the repo. And even if you were okay with specifying the repo, then it would be far simpler to use [specificRepos](https://github.com/probablykasper/readme-template-action#list-specific-repos) for that (like what I've done on [my profile readme](https://github.com/probablykasper/probablykasper)).

I think this is a sensible addition though.

Instead of adding a `repository` input, it looks like you can use the [`GITHUB_REPOSITORY`](https://docs.github.com/en/actions/reference/environment-variables#default-environment-variables) environment variable in the action. That would be a better solution imo

**Comment by hlinero** [comment]
Thanks for the suggestion @probablykasper . I was not aware of the env variables and it definitely a cleaner approach.

I have updated the code to use the default env variables instead.

File: README.md [hunk 1]
```diff
@@ -8,11 +8,11 @@ By default, it takes `TEMPLATE.md` and outputs `README.md`.
 
 ## Inputs
 
-| name         | type   | default         | description |
-| ------------ | ------ | --------------- | ----------- |
-| token        | string |                 | GitHub personal access token used to fetch data. Pass a secret by for instance using `${{ secrets.README_TEMPLATE_TOKEN }}`. [Go here](https://github.com/settings/tokens/new?scopes=read:user) to generate one with the `read:user` scope
-| template     | string | `"TEMPLATE.md"` | Template file path
-| readme       | string | `"README.md"`   | Output file path
+| name         | required | type   | default         | description |
```
Comment (hlinero) [inline README.md:11]: I think it is important to mention that these params are mandatory. 

An alternative to adding a column called `required` could be to just write a sentence saying that they are required. I am opened to suggestions.

Comment (kasper9n) [reply README.md:11]: Sure, that works

**Comment by probablykasper** [comment]
Looks good, thanks for the PR!

**Comment by probablykasper** [comment]
Tested successfully in https://github.com/probablykasper/readme-template-action-test/commit/9b563167da43f0701cd214a3a43b3e681fb455f5

**Pull request merged**